portfolio.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Iryna's Portfolio site</title>
+  <meta charset=“utf-8”>
+  <meta lang="en">
+  <meta name="viewport", content="width=device-width, initial-scale=1">
+  <!-- Below comes the link to Google fonts.-->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+  <!-- Below is the link to my stylesheet.-->
+  <link rel="stylesheet" type="text/css" href="main.css">
+ </head>
+
+<body>
+
+  <div class="grid-container">
+    <!--Below is my own logo designed through a free logo generator on the web.-->
+    <div class="grid-item1"><img src="Iryna_Logo_3.png" alt="logo"/></div>
+    <div class="grid-item2"><h1>Feisty crew</h1> <h2>Of Google Font enthusiasts</h2></div>
+    <div class="grid-item3"><hr></div>
+  </div>
+
+  <section>
+    <!--This container is hosting my nav on the left of the page.-->
+  <div class="flex-container">
+    <a href="#" class="center-image"><img src="grumpy-wizzards.png" alt="fonts"/></a>
+    <a href="#" class="item1" alt="Our Story">Our Story</a>
+    <a href="#" class="item2" alt="Offerings">Fonts We Love</a>
+    <a href="#" class="item3" alt="Testimonials">Fonts we loath</a>
+  </div>
+  </section>
+
+  <h2 class="featured-work">Featured Work of Fonts we Didn't Make</h2>
+<!--Footer shelters 3 tiles and text below them that is wrapped in caption elements.-->
+  <footer class="featured-flex-container">
+    <figure style="margin: 0px; text-align: center;;">
+      <a href="#" class="tile1"><img src="best-google-fonts.png" alt="best-google-fonts-tile" style="max-width: 320px; max-height: 250px; width: 100%; height: 100%; margin-right: 20px;"/></a>
+      <figcaption class="caption">The Best Fonts<br><a href="#">www.fonts.com/best</a></figcaption>
+    </figure>
+    <figure style="margin: 0px; text-align: center;">
+      <a href="#" class="tile2"><img src="ubuntu-font.png" alt="ubuntu-font-tile" style="max-width: 320px; max-height: 250px; width: 100%; height: 100%; margin-right: 20px;"/></a>
+      <figcaption class="caption">Less Used Fonts<br><a href="#">www.fonts.com/less-used</a></figcaption>
+    </figure>
+    <figure style="margin: 0px; text-align: center;">
+      <a href="#" class="tile3"><img src="moon-font.png" alt="moon-font-tile" style="max-width: 320px; max-height: 250px; width: 100%; height: 100%;"/></a>
```

Comment:
For these `<img>` tags we can take all the inline `style` attributes out of the HTML and put them in a css style, usually through a class, like `.tiled-image` or something like that